tests/test_cli_get_set.py

```diff
@@ -192,15 +193,17 @@ def test_set_append(self):
             f.write('''network:
   version: 2
   ethernets:
+    ens2: {dhcp4: yes}
```

Comment:
nitpick (non-blocking): This looks unrelated and unused, why was it introduced?

Reply:
Probably a leftover of some debugging session or another. I'll drop it.